.github/workflows/e2e-nvidia-a10g-x1.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           mode: start
           github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
           ec2-image-id: ami-00c51d9c1374eda97
-          ec2-instance-type: g5.2xlarge
+          ec2-instance-type: g5.4xlarge
```

Comment:
I switched to 4x rather than 2x because I needed more RAM, this has the same amount of GPUs and vRAM as the 2x.